docs/cloud/dbaas/cli.md

```diff
@@ -123,7 +123,7 @@ openstack database user create $INSTANCE_ID $USER_NAME $PASSWORD --databases $DA
 Give database access to an existing user.
```

Comment:
I think we should remove the `access` commands from `cli.md` and move it to `postgresql.md` and mention the schema specifically.